mappings/net/minecraft/client/Mouse.mapping

```diff
@@ -33,30 +34,30 @@ CLASS net/minecraft/unmapped/C_zgebrxop net/minecraft/client/Mouse
 		COMMENT Called when a mouse is used to scroll.
 		ARG 1 window
 			COMMENT the window handle
-		ARG 3 horizontal
+		ARG 3 scrollDeltaX
 			COMMENT the horizontal scroll distance
```

Comment:
```suggestion
			COMMENT the horizontal scroll delta
```

Reply:
Huh, I swear that I deleted that particular javadoc; I thought that it was unnecessary with the new variable names